addons/mail/static/src/models/emoji_category.js

```diff
@@ -34,5 +44,11 @@ registerModel({
             inverse: 'allCategories',
             required: true,
         }),
+        emojiRegistryAsActive: one('EmojiRegistry', {
+            inverse: 'activeCategory',
+        }),
+        isActive: attr({
+            compute: '_computeIsActive',
+        }),
```

Comment:
should be on category view and its parent view, not global on the category itself